server/lib/enrich/video.js

```diff
@@ -15,7 +15,10 @@ module.exports = exports = function video(content, contract, graphicSyndicationF
 	content.extension = DOWNLOAD_ARCHIVE_EXTENSION;
 
 	content.download = Array.from(content.attachments).reverse().find(item => item.mediaType === DOWNLOAD_MEDIA_TYPES.video);
-	content.download.extension = (content.download) ? mime.extension(content.download.mediaType) : '';
+
+	if (content.download) {
+		content.download.extension = (content.download) ? mime.extension(content.download.mediaType) : '';
```

Comment:
As this has moved into a condition we can guarantee that `content.download` is truthy, so the ternary is redundant now:

```suggestion
		content.download.extension = mime.extension(content.download.mediaType);
```